EIPS/eip-draft_location-metadata-standard.md

```diff
@@ -0,0 +1,189 @@
+---
+eip: <to be assigned>
+title: Location Metadata Standard in ERC-721 Tokens
+author:
+Ricardo Chacon (@RicardoChacon)
+Isabella Costanza (@isabellacostanza)
+Gabriela Garro (@gabygarro)
+James Jose (@jamesjosegdsey)
+David Mendes (@de-mendes)
+discussions-to: https://ethereum-magicians.org/t/request-for-feedback-standardizing-location-metadata-and-other-metadata-in-erc-721-tokens/5985
+status: Draft
+type: Informational
+created: 2021-05-04
+requires: 721
+---
+
+## Simple Summary
+
+Standard for location metadata in the ERC-721 tokens. Tokens wishing to include physical location metadata should follow the same standard.
```

Comment:
```suggestion
Standard for location metadata in the ERC-721 tokens. Tokens wishing to include physical location metadata should follow the same standard.
```

I think this could be more concise.

Reply:
Rewritten, please do advise if it needs further changes.